ppr-ui/src/components/common/ButtonFooter.vue

```diff
@@ -209,7 +209,10 @@ export default defineComponent({
     }
     const handleDialogResp = (val: boolean) => {
       localState.showCancelDialog = false
-      if (!val) goToDashboard()
+      if (!val) {
+        emit('cancelProceed')
```

Comment:
So we need to reset validation only when Cancel Dialog is confirmed? What about page refresh, or any other scenarios, would it be reset by itself?

Reply:
This would reset naturally i believe, yes. 
Since we app would be re-mount the state (and subsequently validation state) would be back to the default states it is initialized with.

Reply:
Thanks!